Automattic/vip-go-mu-plugins
# Search: Add hard limit on command for setting up multiple indexes at …

## Description
Since multisites will no longer automatically create indexes on subsite creation per #2690, we should add a hard limit on the CLI command to prevent sites with over 100 blogs automatically creating indexes via the `--network-wide` flag. This will ensure that indexes are purposefully being created after a certain amount.

Added a `--blog-ids` parameter to pass in as an alternative as well.

## Changelog Description
### Plugin Updated: Enterprise Search
Ensure multisites with over 100 subsites cannot automatically create indexes by CLI.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test
To test hard limit:
1) Create multisite with over 100 blogs
2) Attempt to set up indexes all at once via `wp vip-search index --setup --network-wide`
3) Expect it to halt with error

To test new blog-ids flag:
1) Attempt to index 2-3 blogs via `wp vip-search index --setup --blog-ids=1,2,3`
